app/experimenter/docs/swagger-ui.html

```diff
@@ -1350,6 +1350,31 @@
         ]
       }
     },
+    "/api/v5/csv/": {
```

Comment:
I know the old version called this `/csv`, but I think its purpose might be a little more clear if we call it `report` or `reporting` 🤔 what do you think?

Reply:
just to keep consistent with the existing structure, I kept `csv` :)